packages/cactus-plugin-ledger-connector-besu/package.json

```diff
@@ -57,6 +57,7 @@
     "@hyperledger/cactus-core": "1.0.0",
     "@hyperledger/cactus-core-api": "1.0.0",
     "axios": "0.21.4",
+    "amqp-ts": "1.8.0",
```

Comment:
@RafaelAPB I'd keep amqp specific things out of the connectors and instead use an RxJS subject/observable instead (same thing we were talking about a while back during one of the pair programming sessions).
I can show you more detailed code examples just let me know.